airbyte-webapp/src/locales/en.json

```diff
@@ -153,6 +153,7 @@
   "connectionForm.nonBreakingChangesPreference.label": "Non-breaking schema updates detected*",
   "connectionForm.nonBreakingChangesPreference.message": "Set how Airbyte handles syncs when it detects a non-breaking schema change in the source",
   "connectionForm.nonBreakingChangesPreference.ignore": "Ignore",
+  "connectionForm.nonBreakingChangesPreference.disable": "Disable connection",
```

Comment:
I made this "Disable connection" instead of "Pause sync" as shown in the design because we don't really have a concept of pausing in the UI and this is what will actually happen - @andyjih thoughts?